src/utils/provideLocalStorageDataClass.ts

```diff
@@ -61,10 +69,38 @@ export function provideLocalStorageDataClass(className: string) {
     },
   })
 
+  async function initializeContent(
+    initialContent: { _id: string; [key: string]: unknown }[],
+  ) {
+    const initializedKey = `${recordKey}-initialized`
+    const initializedValue = `yes - already initialized with ${await sha1(
+      JSON.stringify(initialContent),
+    )}`
```

Comment:
* [ ] Given the needed local storage is proportional to the number of records anyway, we could simplify the code by using `JSON.stringify(initialContent)` directly as the `initializedValue`.